apps/kg/partition.py

```diff
@@ -2,11 +2,51 @@
 import scipy as sp
 import numpy as np
 import argparse
-import signal
+import os
 import dgl
 from dgl import backend as F
 from dgl.data.utils import load_graphs, save_graphs
 
+def write_txt_graph(path, file_name, part_dict, total_nodes):
```

Comment:
how long does it take to write the graph as a text file?

Reply:
For freebase, it is about 10 minutes.